packages/payment-detection/test/erc777/superfluid-retriever.test.ts

```diff
@@ -99,6 +99,124 @@ const testSuiteWithDaix = (network: string, fDAIxToken: string) => {
         );
       });
     });
+
+    describe('with one off requests payment', () => {
+      it(`should get payment events from SuperFluid via subgraph only one off request payments on ${network}`, async () => {
+        const paymentData = {
+          reference: '0xbeefaccc470c7dbd54de69',
+          txHash: '0xc331a269515c27836051cc4618097f5f1a1c37f79dcb975361022fe3ecfb5c00',
+          from: '0x9c040e2d6fd83a8b35069aa7154b69674961e0f7',
+          to: '0x52e5bcfa46393894afcfe6cd98a6761fa692c594',
+          network: network,
+          salt: '0ee84db293a752c6',
+          amount: '92592592592592000',
+          requestId: '0188791633ff0ec72a7dbdefb886d2db6cccfa98287320839c2f173c7a4e3ce7e1',
+          block: 9806850,
+          token: fDAIxToken,
+        };
+        graphql.request.mockResolvedValue(mockSuperfluidSubgraph[3]);
+
+        const paymentReference = PaymentReferenceCalculator.calculate(
+          paymentData.requestId,
+          paymentData.salt,
+          paymentData.to,
+        );
+        const subgraphReference = `0xbeefac${paymentReference}`;
+        expect(subgraphReference).toEqual(paymentData.reference);
+
+        const graphRetriever = new SuperFluidInfoRetriever(
+          paymentReference,
+          paymentData.token,
+          paymentData.to,
+          PaymentTypes.EVENTS_NAMES.PAYMENT,
+          paymentData.network,
+        );
+        const transferEvents = await graphRetriever.getTransferEvents();
+        expect(transferEvents).toHaveLength(4);
+        expect(transferEvents[0].amount).toEqual('5');
+        expect(transferEvents[1].amount).toEqual('10');
+        expect(transferEvents[2].amount).toEqual('15');
+        expect(transferEvents[3].amount).toEqual('20');
+        expect(transferEvents[0].name).toEqual('payment');
+        expect(transferEvents[0].parameters?.to).toEqual(paymentData.to);
+        expect(transferEvents[0].parameters?.txHash).toEqual(paymentData.txHash);
+        expect(transferEvents[0].parameters?.block).toEqual(paymentData.block);
+        expect(transferEvents[0].parameters?.streamEventName).toEqual(
+          PaymentTypes.STREAM_EVENT_NAMES.END_STREAM,
+        );
+        expect(transferEvents[1].parameters?.streamEventName).toEqual(
+          PaymentTypes.STREAM_EVENT_NAMES.END_STREAM,
+        );
+        expect(transferEvents[2].parameters?.streamEventName).toEqual(
+          PaymentTypes.STREAM_EVENT_NAMES.END_STREAM,
+        );
+        expect(transferEvents[3].parameters?.streamEventName).toEqual(
+          PaymentTypes.STREAM_EVENT_NAMES.END_STREAM,
+        );
+      });
+
+      it(`should get payment events from SuperFluid via subgraph with one-off request payments on ${network} - 1/2`, async () => {
```

Comment:
Why `1/2` ?

Reply:
I had another test which was unecessary and I removed it. Forgot to update the test name.